src/main/java/com/cleanroommc/groovyscript/core/mixin/LoaderControllerMixin.java

```diff
@@ -17,4 +17,11 @@ public void preInit(LoaderState state, Object[] eventData, CallbackInfo ci) {
             GroovyScript.initializeGroovyPreInit();
         }
     }
+
+    @Inject(method = "distributeStateMessage(Lnet/minecraftforge/fml/common/LoaderState;[Ljava/lang/Object;)V", at = @At("RETURN"))
+    public void postInit(LoaderState state, Object[] eventData, CallbackInfo ci) {
+        if (state == LoaderState.POSTINITIALIZATION) {
+            GroovyScript.initializeGroovyPostInit();
+        }
+    }
```

Comment:
Why do create a new method and not use the exisiting mixin?

Reply:
Because the existing method is at HEAD, which would cause scripts to run before fml post init

Reply:
Then use `LoaderState.AVAILABLE`